easybuild/easyconfigs/y/Yade/Yade-1.20.0-foss-2016a.eb

```diff
@@ -0,0 +1,88 @@
+easyblock = 'CMakeMake'
+
+name = 'Yade'
+version = '1.20.0'
+
+homepage = 'www.yade-dem.org'
+description = """ Yade is an extensible open-source framework for discrete numerical models, 
+ focused on Discrete Element Method. The computation parts are written in c++ using flexible object model, 
+ allowing independent implementation of new alogrithms and interfaces. 
+ Python is used for rapid and concise scene construction, simulation control, postprocessing and debugging. """
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://launchpad.net/yade/trunk/yade-1.00.0/+download/']
+sources = [SOURCELOWER_TAR_GZ]
+
+pyver = '2.7.11'
+versionsuffix = '-Python-%s' % pyver
+
+dependencies = [
+    ('git', '2.8.0'),
+    ('freeglut', '3.0.0'),
+    ('loki', '-------'),
+    ('Boost', '1.60.0'),
+    ('fakeroot', '-------'),
+    ('dpkg', '-------'),
```

Comment:
This is probably not required, since this is a tool for managing packages on Debian?